package.json

```diff
@@ -105,5 +107,11 @@
     "hooks": {
       "commit-msg": "commitlint --env HUSKY_GIT_PARAMS"
     }
+  },
+  "standard-version": {
+    "skip": {
+      "commit": true,
+      "tag": true
```

Comment:
Note: the eventual goal is for standard-version to handle creating the release commits + tags as well, but this requires some more follow-up work.